cognite/gqlpygen/main.py

```diff
@@ -1,60 +1,165 @@
 import importlib
 import importlib.util
 import inspect
+import os
+import subprocess
 import sys
 from pathlib import Path
 
 import click
 import typer
 
+from cognite.dm_clients.config import settings
 from cognite.dm_clients.domain_modeling.schema import Schema
 from cognite.gqlpygen.generator import to_client_sdk
 from cognite.gqlpygen.misc import to_client_name, to_schema_name
 
 app = typer.Typer()
 
 
```

Comment:
Here there should probably be some output to the user on whether the settings have been loaded.

Reply:
🤔  Not a big fan of unsolicited output. How about a CLI command that echoes all loaded settings (and hides `client_secret`)? https://github.com/cognitedata/cognite-gql-pygen/pull/20/commits/e987080b1a07021c8ebef7a0cb85c6d634c647fd

Reply:
I think that is a good idea, but lets save that for future development.